tests/posix/mt/mt.cpp

```diff
@@ -44,13 +44,12 @@ static PMEMfilepool *global_pfp;
 
 class mt : public pmemfile_test {
 public:
-	long ncpus;
-	pthread_t *threads;
+	unsigned ncpus;
+	std::list<std::thread> threads;
```

Comment:
Just one note from me, remember that std::thread is non-copyable so eplace or push rvalue references so they get moved.